shoop/core/models/products.py

```diff
@@ -252,12 +252,40 @@ def get_shop_instance(self, shop):
 
         return shop_inst
 
-    def get_cheapest_child_price(self, context, quantity=1):
+    def get_child_price_range(self, context, quantity=1):
+        """
+        Get the prices for cheapest and the most expensive child
+
+        The attribute used for sorting is `PriceInfo.price`.
+
+        :type context: shoop.core.contexts.PriceTaxContext
+        :type quantity: int
+        :return: a tuple of prices
+        :rtype: (shoop.core.pricing.Price, shoop.core.pricing.Price)
+        """
+        infos = sorted(
+            [c.get_price_info(context, quantity=quantity) for c in self.variation_children.all()],
+            key=lambda x: x.price
+        )
+        return infos[0].price, infos[len(infos)-1].price
```

Comment:
Parens around the rvalue would be nice.

Also, this is Python, you can just do `[-1].price`, you know? ;)